specs/bond-manager.md

```diff
@@ -0,0 +1,146 @@
+# Bond Manager Interface
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [The Bond Problem](#the-bond-problem)
+  - [Simple Bond](#simple-bond)
+  - [Variable Bond](#variable-bond)
+- [Contract Interface](#contract-interface)
+- [Bond Manager Implementation](#bond-manager-implementation)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+In the context of permissionless output proposals, bonds are a value that must
+be attached to an output proposal. In this case, the bond will be paid in ether.
+
+By requiring a bond to be posted with an output proposal, spam and invalid outputs
+are disincentivized. It's important to note that it is directly disincentivized
```

Comment:
> It's important to note that it is directly disincentivized
because if the output is invalid, it will be seized by a set of challenge agents
when the output is deleted.

Its not clear what `it` is in this context, this sentence can be made more clear

Reply:
Cleaned it up a bit :)